### PR TITLE
contributor guide and code of conduct

### PR DESCRIPTION
**Pull request by cpolitano**
first pass at these items just to get them out there!

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+## Why contribute to Hylo?
+
+Hylo is a co-created and community-led platform. As an open-source project (as opposed to closed-source, proprietary technology), everyone is free to look at the codebase and add to it, and we don't charge people to use the platform. Hylo doesn't exist to maximize revenue for anyone. Instead, Hylo's purpose is to help groups organize impactful action.
+
+Hylo relies on donations - of money, and of time and energy from people who share a vision for a world where our collective resources are shared as a commons. Our water, our land, and even our technology tools can be cooperatively stewarded by the community, for the benefit of all.
+
+Do you dream of a world that works for all? If so, Hylo could benefit from your care. Here’s how to get involved.
+
+## Our Culture & Design Principles
+As the builders of a collaboration platform, we strive for helpful and respectful interactions. Our culture is kind and encouraging, and it’s important to us that the community creating Hylo is a safe and welcoming space for all. We are incorporating the [Prosocial](https://www.prosocial.world/) framework into our processes to evolve our team coherence even more.
```
Comment (tibetsprague) [inline CONTRIBUTING.md:12]: link to code of conduct?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+## How to help
+There are so many ways to contribute to Hylo, and we’d be delighted for you to participate in whatever way is most exciting to you! Of course, as a software project, we are always welcoming volunteer coders. But you don’t have to contribute code to be helpful! Regardless of your technical chops, there’s plenty to do.
+
+### Here are some important ways to help out:
+- Beta test new releases to ensure high quality functionality
+- Create documentation, such as onboarding materials to help communities thrive on Hylo
+- Submit feedback, bug reports, and feature requests
+- Contribute code!
+
+To get involved in beta testing, creating onboarding materials, or offering feedback on product ideas or designs, join [Building Hylo](https://www.hylo.com/groups/building-hylo/join/RcDe6vcG7u).
```
Comment (tibetsprague) [inline CONTRIBUTING.md:48]: a little nervous about making this invite public... but ok

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+We accept bug reports through Intercom within the Hylo web application.
+
+We accept [feature requests here](https://airtable.com/shrAWV0UkgDoM55TW). Follow along on our transparent [development roadmap here](https://airtable.com/shrcTCuwNDVTS1DpE).
```
Comment (tibetsprague) [inline CONTRIBUTING.md:52]: is this really true? are we checking this?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+### Our dev process
+Hylo is stewarded by a team of engineers, designers, artists, and product managers from [Terran Collective](https://www.terran.io/). We create project scopes in partnership with organizations who use Hylo and have resources to share to help improve the platform for everyone. These project scopes guide us to set a timeline and assemble a team to carry out the development in co-creation with our partners. Right now, we meet weekly for collaborative sprint planning and backlog generation. We also like to pair program, and have deep dive product conversations. If you’d like to be a part of it, here’s what to do:
+
+### Steps to get involved:
+- Join the [Building Hylo](https://www.hylo.com/groups/building-hylo/join/RcDe6vcG7u) group, our open source community for people co-creating Hylo!
+- Reach out to Hylo's tech lead Tibet Sprague to say hi: tibet [at] terran [dot] io
```
Comment (tibetsprague) [inline CONTRIBUTING.md:64]: lets use tibet@hylo.com

File: CONTRIBUTING.md (hunk 2 above)
Comment (cpolitano) [reply CONTRIBUTING.md:48]: what's your alternative suggestion for people to get involved?

File: CONTRIBUTING.md (hunk 3 above)
Comment (cpolitano) [reply CONTRIBUTING.md:52]: I'm not sure what you mean, but this type of feedback ("is this really true?") doesn't feel good for me to receive. this is the only place these things exist for now. happy to update later when i get a gitbook up and running. do you have an alternate suggestion?

File: CONTRIBUTING.md (hunk 4 above)
Comment (cpolitano) [reply CONTRIBUTING.md:64]: ok

File: CONTRIBUTING.md (hunk 1 above)
Comment (cpolitano) [reply CONTRIBUTING.md:12]: yes

File: CONTRIBUTING.md (hunk 2 above)
Comment (tibetsprague) [reply CONTRIBUTING.md:48]: nothing right now

**Pull request merged**